_posts/en/newsletters/2021-09-15-newsletter.md

```diff
@@ -0,0 +1,209 @@
+      might require creating multiple transactions that affect multiple
+      other users.  TLUV allows any single member to exit at any time
+      without requiring anything be presigned or affecting any other
+      users, beyond revealing that one member has left the joinpool.
+
+    - **Merkle tree tweak:** taproot addresses can also commit to the
+      merkle root for a tree of tapscripts, and it'll be one of those
+      tapscripts in the transaction input that will be executing the
+      TLUV opcode.  TLUV allows that script to specify how that part of
+      the merkle tree should be modified.  For example, the currently
+      executed node (tapleaf) can be removed from the tree (e.g. someone
+      exiting a joinpool) or replaced with a tapleaf paying a different
+      tapscript.  TLUV computes the modified merkle root and ensures
+      that is what the same-position output will pay.
+
+      Towns's email describes how this can be used to implement Bryan
+      Bishop's 2019 [vault][topic vaults] design (see [Newsletter
+      #59][news59 vaults]).  Alice creates two keypairs, one for her
+      less secure hot wallet and one for her more secure cold wallet.
+      The cold wallet key becomes the taproot internal key, allowing it
+      to spend the funds any time.  The hot wallet key is used with TLUV
+      to only allow spending to a modification of the merkle tree that
+      includes a time delay before a second spend from the hot wallet
+      key may be sent.
+
+      That means Alice can start a spend of all her funds with her hot
+      key, but she has to create an onchain transaction for that and
+      then wait for the time delay (e.g.  1 day) to complete before she
+      can truly spend her funds to an arbitrary address.  If someone
+      else is using Alice's hot key to start the spending process, Alice
+      can use her cold key to move the funds to a safe address.
+
+    - **Amount introspection:** in addition to TLUV, a second opcode
+      would be added that pushes to the script execution stack the
+      bitcoin value of the input and its corresponding output.  This
+      allows math and comparison opcodes to restrict the amount that can
+      be spent.
+
+      In the case of a joinpool, this would be used to ensure a
+      departing member could only withdraw their own funds.  In a vault,
+      this could be used to set a periodic withdrawal limit, e.g. 1 BTC
+      per day.
+
+    As of this writing, the proposal was still receiving initial
+    feedback on the mailing list.  We'll summarize any notable comments
+    in a future newsletter.
+
+- **Signet reorg discussion:** developer 0xB10C [posted][b10c post] a
+  proposal to the Bitcoin-Dev mailing list for creating periodic block
+  chain reorganizations (reorgs) on [signet][topic signet].  The blocks
+  that would be eventually reorged would signal on one of the
+  versionbits, allowing anyone who didn't want to track reorgs to ignore
```

Comment:
Should this be "version bits" (i.e. one of the bits in the version field of the header)? To my mind, "versionbits" refers specifically to the BIP9/8 signaling mechanism (and even there, the BIP seems to use it interchangeably with "version bits" https://github.com/bitcoin/bips/blob/bd943663d6908679ca4f94276589ef402f865739/bip-0009.mediawiki)